codecov.yml

```diff
@@ -0,0 +1,12 @@
+coverage:
+  status:
+    project:
+      default:
+        enabled: yes
+        target: auto
```

Comment:
@e-kolpakov I'm not sure what the "auto" value of `target` means here -- in the example provided by @rlucioni there was an integer value for this field.